web-react-ts/src/pages/services/graphs/TrendsMenu.tsx

```diff
@@ -28,7 +28,7 @@ const TrendsMenu = () => {
             iconComponent={Charts}
             title="Graphs"
             color="members"
-            onClick={() => navigate(`/${churchType.toLowerCase()}/graphs`)}
+            onClick={() => navigate(`/${churchType?.toLowerCase()}/graphs`)}
```

Comment:
If there is no churchType, the link will lead to a 404. Are you able to understand why there would be no churchType?